docs/fi/faq.md

```diff
@@ -19,58 +32,72 @@ tulokset napin taakse, jotta emme tee mitään ilman suostumustasi ja että
+huolehtimaan tietoturvasta. Yritykset [voivat myös ottaa omia suojattavia kohteitaan](https://hygiene.badrap.io/company/) seurantaan
+automaattisesti, erillisten Badrap Appien avulla.
+
+Ilmaispalvelu lisää meidän tunnettuvuutta ja on yksi keino muiden joukossa poistaa kaikki tietoturvatiedon välittämiseen liittyvät esteet.
 
 ## Mistä tietoturvatutkijoiden varoitukset tulevat?
 
-Olemme aloittaneet pyytämällä dataa muutamasta meille ennestään tutusta
+Aloittaneet pyytämällä dataa muutamasta meille ennestään tutusta
 äärimmäisen laadukkaasta ja luotettavasta lähteestä, joita käyttävät myös monet
-muut tietoturvasta ja haavoittuvuushavainnoista kiinnostuneet tahot. Etsimme
-koko ajan lisää laadukkaita haavoittuvuustiedon lähteitä. Tämänhetkisiä
-partnereitamme ovat:
+muut tietoturvasta ja haavoittuvuushavainnoista kiinnostuneet tahot.
+
+Ajantasainen lista tietoturvatutkijoista löytyy osoitteesta <https://badrap.io/researchers>
+
+
+## Minkälaista tietoa Badrapistä saa?
+
+Meillä on kolmea erityyppistä tietoturvatietoa:
 
-- [Shadowserver](https://www.shadowserver.org/)
-- [Fitsec](https://www.fitsec.com/)
-- Telia
+* Tietovuodot - ovatko Internetin palvelut vuotaneet sinun henkilökohtaisia tietoja
+* Haavoittuvuudet - onko IP-osoitteessasi, josta kytkeydyt Internettiin havaittu haavoja, jotka kannattaisi korjata ennen kuin joku käyttää niitä hyväksi.
+* Hyökkäykset - joskus voit olla hyökkääjä tietämättäsi. Esimerkiksi, jos koneellasi sattuisi olemaan haittaohjelma, se voisi etsiä aktiivisesti uusia uhreja. Olet itsekin uhri, mutta uuden uhrin näkökulmasta hyökkääjä. Firman nimi tulee muuten tästä: Badrap - ansaitsematon huono maine, kuten [Merriam-Webster](https://www.merriam-webster.com/words-at-play/usage-bad-rap-vs-bad-rep-vs-bad-wrap) termin määrittää.
 
 ## Seuraatteko minua tai verkkojani?
 
 Lyhyt vastaus: emme.
 
 Pidempi vastaus löytyy [tietosuojaselosteestamme](./privacy.md).
 Teemme kävijätilastoihin pohjautuvaa palvelun kehittämistä, kuten kahdeksan biljoonaa
-muutakin palvelua. Sillä erotuksella tosin, että mietimme jatkuvasti miten pitäisimme
-yksityisyydestäsi entistäkin paremmin huolta.
+muutakin palvelua. Mutta käyttäjiemme yksityisyys on meille tärkeää. Oikeasti! Tämä ei
+ollut kopioitu korulause.
 
-Demonstroidaanpa kuinka tärkeä asia meille on. Käytämme tilastointiin Google
-Analytics -työkalua, kuten miljoonat muut. Tilastointi on meille tärkeää,
+Demonstroidaanpa: tilastointi on meille tärkeää,
 jotta voimme kehittää palvelua. Kerromme nyt miten estät tilastoinnin.
 
-Voit estää tilastoinnin käyttämällä selaimessasi ad blocker -lisäosaa.
-Chrome-selaimessa kätevä työkalu on esimerkiksi uBlock Origin. Nyt kun
-kerroimme sinulle tämän, niin pieni pyyntö. Salli valitsemastasi työkalusta
-meidän tekemä tilastointi ja estä kaikki muut. :)
+Voit estää tilastoinnin käyttämällä selaimessasi sisällön estäjää.
+Chrome-selaimessa kätevä työkalu on esimerkiksi uBlock Origin.
```

Comment:
Pitäisikö tähän laittaa vaan suoraan linkit uBlock Originiin Chromelle ja Firefoxille? 🙂

https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm
https://addons.mozilla.org/en-GB/firefox/addon/ublock-origin/

Reply:
Ehkä näin: 

Voit estää tilastoinnin käyttämällä selaimessasi sisällön estäjää.
Esimerkiksi uBlock Origin toimii sekä [Chromessa][https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm]
että [Firefoxissa](https://addons.mozilla.org/en-GB/firefox/addon/ublock-origin/)